packages/app/src/templates/components/templates/templates.component.tsx

```diff
@@ -3,24 +3,71 @@
  * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
  */
 
-import React, { useState } from "react"
+import React, { useState, useEffect } from "react"
 import { FunctionComponent } from "Renderer/types/function-component.interface"
 import { TemplatesProps } from "App/templates/components/templates/templates.interface"
 import { TemplatesSection } from "App/templates/components/templates/templates.styled"
 import { TemplatesPanel } from "App/templates/components/templates-panel"
 import { TemplatesList } from "App/templates/components/templates-list"
 import { TemplateForm } from "App/templates/components/template-form"
 import { NewTemplate } from "App/templates/dto"
-import { TemplateError } from "App/templates/constants"
+import { TemplateError, TemplateDeletingState } from "App/templates/constants"
+import DeleteConfirmationModal from "App/templates/components/delete-confirmation-modal/delete-confirmation-modal.component"
+import { defineMessages } from "react-intl"
+import { intl, textFormatters } from "Renderer/utils/intl"
+import InfoPopup from "App/ui/components/info-popup/info-popup.component"
+import { Message as TranslationMessage } from "Renderer/interfaces/message.interface"
+import ErrorModal from "App/ui/components/error-modal/error-modal.component"
+import DeletingModal from "App/ui/components/deleting-modal/deleting-modal.component"
+
+const messages = defineMessages({
+  deleteModalTitle: { id: "module.templates.deleteModalTitle" },
+  deleteModalBody: { id: "module.templates.deleteModalBody" },
+  templateDeleted: { id: "module.templates.templateDelete" },
+  templatesDeleted: { id: "module.templates.templateDelete" },
+  deletingModalTitle: { id: "module.templates.deletingModalTitle" },
+  deletingModalSubtitle: { id: "module.templates.deletingModalSubtitle" },
+  deleteModalErrorSubtitle: { id: "module.templates.deleteModalErrorSubtitle" },
+})
 
 export const Templates: FunctionComponent<TemplatesProps> = ({
   templates,
   loading,
   error,
+  deletingState,
   createTemplate,
   deleteTemplates,
+  hideDeleteModal,
 }) => {
   const [newTemplateOpen, setNewTemplateOpenState] = useState<boolean>(false)
+  const [deleteModalOpen, setDeleteModalOpen] = useState<boolean>(false)
+  const [deletedTemplates, setDeletedTemplates] = useState<string[]>([])
+
+  useEffect(() => {
+    if (deletingState === TemplateDeletingState.Success) {
+      const timeout = setTimeout(() => {
+        setDeletedTemplates([])
+        hideDeleteModal()
+      }, 5000)
```

Comment:
Just curious and asking - is there any reason why exactly 5000?

Reply:
Yes, according to acceptance criteria popup should disappear after 5 seconds

Reply:
I added variable with time in [7c222ee](https://github.com/mudita/mudita-center/pull/764/commits/7c222eed5a12a682f3d9b7d9ec91c06d8a7edc54)